src/rpc/protocol.cpp

```diff
@@ -114,13 +114,3 @@ bool GetAuthCookie(std::string *cookie_out)
         *cookie_out = cookie;
     return true;
 }
-
-void DeleteAuthCookie()
```

Comment:
@laanwj How should I handle `DeleteAuthCookie()` in this PR? I'd suggest keeping this PR as-is and handle the re-introduction of `StopRPC()`/`DeleteAuthCookie()` in a separate PR. Makes sense?